.github/CODEOWNERS

```diff
@@ -0,0 +1,257 @@
+Tasks/AndroidSigningV2/*    @madhurig
+
+Tasks/AndroidSigningV3/*    @madhurig
+
+Tasks/ANTV1/*   @madhurig
+
+Tasks/AppCenterDistributeV1/*   @madhurig
+
+Tasks/AppCenterTestV1/*   @madhurig
+
+Tasks/ArchiveFilesV2/*  @bryanmacfarlane
+
+Tasks/AzureAppServiceManageV0/*     @kmkumaran      @SumiranAgg     @vincentdass
+
+Tasks/AzureCLIV1/*      @sachinma
+
+Tasks/AzureCloudPowerShellDeploymentV1/*    @kmkumaran
+
+Tasks/AzureFileCopyV1/*    @kmkumaran
+
+Tasks/AzureFileCopyV2/*    @kmkumaran
+
```

Comment:
The new Azure function and container tasks that we added are missing in the list.

Reply:
added missing task. thanks.